at-date-api/src/main/kotlin/com/amerharb/atdate/app.kt

```diff
@@ -0,0 +1,111 @@
+package com.amerharb.atdate
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import java.util.*
+
+const val PORT = 8000
+
+fun main() {
+    embeddedServer(Netty, port = PORT, module = Application::module).start(wait = true)
+}
+
+fun Application.module() {
```

Comment:
doc what does this do?

Reply:
you mean line 17?
no need to doc this it is the main module